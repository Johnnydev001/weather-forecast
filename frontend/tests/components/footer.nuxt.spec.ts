import { mountSuspended } from "@nuxt/test-utils/runtime"
import footerComponent from "~/components/footer/footer.vue"
import {describe, it, expect } from 'vitest'

describe('Footer component tests', () => {
    it('should match the footer component snapshot', async () => {
        const footer = await mountSuspended(footerComponent)
    
        expect(footer.text()).toMatchSnapshot(`
            <p class="copyright">&copy; 
                <a href="https://www.linkedin.com/in/jo%C3%A3o-saraiva-ab662b197/">
                    Jo√£o Saraiva - 2024
                </a>
            </p>   
            `)
    })
})
