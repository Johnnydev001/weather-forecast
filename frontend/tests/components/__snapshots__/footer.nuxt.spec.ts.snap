// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Footer component tests > should match the footer component snapshot > 
            <p class="copyright">&copy; 
                <a href="https://www.linkedin.com/in/jo%C3%A3o-saraiva-ab662b197/">
                    João Saraiva - 2024
                </a>
            </p>   
             1`] = `"©  João Saraiva - 2024"`;
